Rails 4 Support

irubnich opened the pull request.

pjambet commented on .travis.yml:
```diff
@@ -2,4 +2,6 @@ sudo: false
\ No newline at end of file
+  - 2.1.7
+  - 2.2.2
+script: bundle exec rspec spec
```
would be great to add rails 3.2.x and 4.2.x (and maybe 4.0.x/4.1.x who knows?!) to the test matrix

pjambet commented:
Looks good! Just one comment about the travis config.

irubnich commented on lib/split_cacheable.rb:
```diff
@@ -1,6 +1,6 @@
 require "split_cacheable/version"
 require 'split_cacheable/helper'
-require 'split_cacheable/engine' if defined?(Rails) && Rails::VERSION::MAJOR === 3
+require 'split_cacheable/engine' if defined?(Rails) && [3, 4].include?(Rails::VERSION::MAJOR)
```
This must only be 4

irubnich commented:
@pjambet I adjusted this so it supports both 3.2 and rails 4 and updated the readme with the info. It's just the difference of adding or not adding the Rails 4 gem.

Version bump is now only to 1.1.0.

irubnich commented on lib/split_cacheable.rb:
```diff
@@ -1,6 +1,8 @@
 require "split_cacheable/version"
 require 'split_cacheable/helper'
-require 'split_cacheable/engine' if defined?(Rails) && Rails::VERSION::MAJOR === 3
+if defined?(Rails) && [3, 4].include?(Rails::VERSION::MAJOR) && ActionController::Base.method_defined?(:fragment_cache_key)
```
@pjambet this check should completely ensure that split_cacheable has what it needs

irubnich commented on README.md:
```diff
@@ -16,13 +16,23 @@ Then run:
+
+Because this gem uses action caching, a feature removed from Rails 4, you will also need to include the `protected_attributes` gem in your Gemfile:
+
+    gem 'protected_attributes'
```
Wrong gem, I'm an idiot

irubnich commented:
@pjambet plz

pjambet commented:
:+1:

irubnich commented:
@danielschwartz can you give me push permissions for this gem on rubygems? Would love to push out this new version (not urgent)
